### PR TITLE
HW5

### PR DESCRIPTION
**Pull request by artemVash**
Screenshot_20210423_154020_by vashkevich lesson1hw
Screenshot_20210423_154112_by vashkevich lesson1hw

### CODE REVIEW

**Comment by artemVash** [comment]
Screenshot_20210423_162230_by vashkevich lesson1hw
Screenshot_20210423_162241_by vashkevich lesson1hw
Screenshot_20210423_162247_by vashkevich lesson1hw

**Comment by artemVash** [comment]
Screenshot_20210423_162230_by vashkevich lesson1hw
Screenshot_20210423_163131_by vashkevich lesson1hw
Screenshot_20210423_163301_by vashkevich lesson1hw

**Review by e-kleban — APPROVED** [review]

**Review by YarikGuk — APPROVED** [review]

**Review by Berikz80 — APPROVED** [review]

File: app/src/main/java/HW5/HomeWork5Activity.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        val password = findViewById<TextInputEditText>(R.id.input_password)
+        val btnRegister = findViewById<Button>(R.id.button_register)
+
+        val nameLayout = findViewById<TextInputLayout>(R.id.name_layout)
+        val passwordLayout = findViewById<TextInputLayout>(R.id.password_layout)
+
+        btnRegister.setOnClickListener {
+
+            if (nameUser.text.isNullOrEmpty()) {
+                nameLayout.error = "Поле пустое"
```
Comment (TMSlebedev) [inline app/src/main/java/HW5/HomeWork5Activity.kt:29]: выноси в стринги

**Review by TMSlebedev — APPROVED** [review]

**Pull request merged**